javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/TracerInstaller.java

```diff
@@ -165,6 +191,14 @@ private static void installExporter(SpanExporterFactory spanExporterFactory, Pro
     log.info("Installed span exporter: " + spanExporter.getClass().getName());
   }
 
+  private static void installCollector(
```

Comment:
What collector? I see `MetricProducerCustomizer` and `MetricProducer` in method body, what is collector? :)

Reply:
This is a leftover from the previous naming. I will rename it to match the new name.